providers/base/units/stress/suspend_cycles_reboot.md

```diff
@@ -0,0 +1,96 @@
+<!-- markdownlint-disable MD033 -->
+# Flow of the suspend-cycles-stress-test test plan
+
+This description will focus on the suspend cycles and reboot process
+The  remaining work log check, suspend time check, and log attachments  will be executed at the end of a suspend and reboot jobs.
+
+## Definition of the test case name
+
+- **suspend\_cycles\_{n}\_reboot{k} :** Indicates the execution of a suspend operation, n is the suspend index of the k<sup>th</sup> round of reboot. Simply put: S<sub>k</sub>n
+- **suspend\_cycles\_reboot{k}:** Indicate the execution of a reboot operation, k is the the reboot index. Simply put: R<sub>k</sub>
+
+## For Instance
+
+If do 5 suspends per reboot round and do 3 rounds (N = 5, K=3), it means:
+> N: numbers of suspend  in each reboot
+>
+> K: numbers of reboot
+
+- suspend\_cycles\_1\_reboot1: S<sub>A</sub>1
+- suspend\_cycles_1\_reboot{{suspend\_reboot\_id}}: S<sub>k</sub>1 (k: from A to C)
+- suspend\_cycles\_{{suspend\_id}}\_reboot{{suspend\_reboot\_id}}: S<sub>k</sub>n (n: from 2 to 5, k: from A to C)
+- suspend\_cycles\_reboot{{suspend\_reboot\_id}}: R<sub>k</sub> (k: from A to C)
+
+The flow will be the following:
+
+S<sub>A</sub>1 &rarr; S<sub>A</sub>2 &rarr; S<sub>A</sub>3 &rarr; S<sub>A</sub>4 &rarr; S<sub>A</sub>5 &rarr; R<sub>A</sub>
+
+&rarr; S<sub>B</sub>1 &rarr; S<sub>B</sub>2 &rarr; S<sub>B</sub>3 &rarr; S<sub>B</sub>4 &rarr; S<sub>B</sub>5 &rarr; R<sub>B</sub>
+
+&rarr; S<sub>C</sub>1 &rarr; S<sub>C</sub>2 &rarr; S<sub>C</sub>3 &rarr; S<sub>C</sub>4 &rarr; S<sub>C</sub>5 &rarr; R<sub>C</sub>
+
+## The relateion between template job and the resources job
+
+- suspend\_cycles\_1\_reboot1 : job
+  - ie : S<sub>A</sub>1
```

Comment:
```suggestion
  - For example: S<sub>A</sub>1
```

"i.e." means "id est" in Latin, it's different from an example, so I think it's better to just write "For example" or "For instance" in plain text :)

(This applies to other lines below, too)